### PR TITLE
Nuget

### PR DESCRIPTION
**Pull request by KevinLaMS**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/227)

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Validation has completed

**Comment by rrelyea** [comment]
Despite merging, nuget cli doesn't seem to be in manifests\Microsoft at all.

Reason why? or some problem caused it?

@aortiz-msft @anangaur @chgill-msft

File: manifests/Microsoft/NuGet/5.5.1.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Id: Microsoft.NuGet
+Version: 5.5.1
+Name: NuGet
+Publisher: Microsoft Corporation
+Homepage: https://www.nuget.org/
+License: © Microsoft Corporation. All rights reserved.
+Description: NuGet Command Line
+Installers: 
+    - Arch: x86
+      Url: https://dist.nuget.org/win-x86-commandline/latest/nuget.exe
```
Comment (nkolev92) [inline manifests/Microsoft/NuGet/5.5.1.yaml:10]: This is unfortunately a fragile implementation. 

See https://docs.microsoft.com/en-us/nuget/api/tools-json for details. 

The latest link will point to a new version soon enough.

**Comment by nkolev92** [comment]
@rrelyea it was reverted in https://github.com/microsoft/winget-pkgs/pull/248 :)

**Pull request merged**